themes/red-theme/lms/static/sass/partials/bootstrap/_theme.scss

```diff
@@ -0,0 +1,13 @@
+// Sample red theme to demonstrate SASS overrides
```

Comment:
I'm thinking you should make the simple change to the non-Bootstrap red theme Sass so that it styles the same way. However,  as you mentioned, this will be difficult to do, since the old Sass was not designed to be themed. What do you think? Could you do a simple version that at least makes a few things red? At least, not using the red background color would make the pages consistent.